Webapp/sources/blueprints/reaction_table/routes.py

```diff
@@ -4,19 +4,20 @@
 """
 
 import re
-from datetime import datetime
 from typing import Dict, List, Tuple, Union
 from urllib.parse import quote
 from urllib.request import urlopen
 
 from flask import current_app, jsonify, render_template, request
 from flask_login import login_required
 from rdkit import Chem
+from rdkit.Chem import Descriptors
 from sources import models, services
 from sources.auxiliary import abort_if_user_not_in_workbook, smiles_symbols
 from sources.dto import ReactionNoteSchema
 
 from . import reaction_table_bp
+from Webapp.sources.services.reaction_classification import classify_reaction
```

Comment:
guessing this is pycharm's refactor->move?  more consistent to use as: services.reaction_classification.classify()